stories/primer/heading_component_stories.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Primer::HeadingComponentStories < ViewComponent::Storybook::Stories
+  layout "storybook_preview"
+
+  story(:heading) do
```

Comment:
I'd like to se a `tag` control, just to showcase different headings (h1, h2, ...)